### ISSUE TITLE
Resources created in incorrect/unspecified namespace

### ISSUE THREAD

**Issue by paulfantom** [timescale/tobs#457]
**What happened?**

Not all resources have namespace specified and thus can be running in `default` namespace.

**Did you expect to see something different?**

I expect all resources to be confined to a specified namespace

**How to reproduce it (as minimally and precisely as possible)**:

1. Run: `helm template test --namespace ns chart/`
2. Check following resources:
```
  Job - test-patroni-jl 
  Pod - opentelemetry-operator-cert-manager-test-connection 
  Pod - opentelemetry-operator-controller-manager-metrics-test-connection 
  Pod - opentelemetry-operator-webhook-test-connection 
  StatefulSet - test-timescaledb
```

**Comment by nhudson** [comment]
This is going to require fixes in the [timescaledb-single](https://github.com/timescale/helm-charts) and [opentelemetry-operator](https://github.com/open-telemetry/opentelemetry-helm-charts) Helm charts

**Comment by nhudson** [comment]
@paulfantom the opentelemetry-operator Pods are only executed when you run Helm tests.  I started working on a PR to opentelemetry-operator but I don't really think it's needed being there is an annotation with `"helm.sh/hook": test`.  This will not get installed when running `helm upgrade` or `helm install`.  

I believe with the merge of timescale/helm-charts#412 this can be closed.  Thoughts?

**Comment by nhudson** [comment]
timescale/tobs#535 is related as it instructs the user to manually delete the `opentelemetrycollector` CR manually.

**Comment by nhudson** [comment]
Created https://github.com/open-telemetry/opentelemetry-helm-charts/pull/318 to fix the otel manifests.

**Comment by nhudson** [comment]
https://github.com/open-telemetry/opentelemetry-helm-charts/pull/318 has been merged and is now fixed in `v0.11.5` of the Helm chart.  

We can close this issue, once we bump the timescaledb-single Helm chart and update tobs.

**Comment by paulfantom** [comment]
> We can close this issue, once we bump the timescaledb-single Helm chart and update tobs.

Let's just merge this as we'll bump those components either way.
